index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <p class="logo">My Logo</p>
+      <img class="burger-menu" src="./img/Enabled.png" alt="burger-menu">
+    </div>
+  </header>
+  <main>
+    <article>
+      <h1 class="title">Hey there. <br> I'm Karim.</h1>
+      <h3 class="sub-title-1">I'm a Software Developer</h3>
+      <p class="infography">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hesitate to contact me.
+      </p>
+    </article>
+  </main>
+  <footer>
+    <div>
+      <img class="social-media-img" src="./img/Social Media.png" alt="">
```

Comment:
- [ ] these icons are meant for your social media links, the whole icons being in a single image will not do what we need it for, Kindly ensure that the list of icons is an `a` tag wrapped in a `li` tag then all wrapped in a `ul` tag.

something like this 

```
<ul>
  <li><a>home</a></li>
  <li><a>house</a></li>
  <li><a>yard</a></li>
  <li><a>apartment</a></li>
</ul>
```

This is good for semantic and user experience reasons respectively and also enables one to connect with the icons using the `href` attribute.

<img width="194" alt="image" src="https://github.com/Rebelzob/Portfolio/assets/41163571/d8459525-ffce-4feb-8bf2-1249c70d8cfe">